api/src/ISceneRunner.cs

```diff
@@ -33,7 +34,19 @@ public interface ISceneRunner : IDisposable
     /// <param name="autofree">If true the loaded scene will be automatic freed when the runner is freed.</param>
     /// <param name="verbose">Prints detailed infos on scene simulation.</param>
     /// <returns></returns>
-    public static ISceneRunner Load(string resourcePath, bool autofree = false, bool verbose = false) => new Core.SceneRunner(resourcePath, autofree, verbose);
+    public static ISceneRunner Load(string resourcePath, bool autofree = false, bool verbose = false)
```

Comment:
This is an interface and should not contain code ;)
Please move this part back to the constructor.